input_validation/module_utils/schema/provision_config.json

```diff
@@ -0,0 +1,146 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "iso_file_path": {
+      "type": "string",
+      "description": "Path of the ISO file to be used for provisioning the clusters"
+    },
+    "node_name": {
+      "type": "string",
+      "description": "Name of the node.",
+      "pattern": "^[a-zA-Z0-9._-]+$"
+    },
+    "domain_name": {
+      "type": "string",
```

Comment:
provision_config.yml will not be having iso_file_path, node_name and switch details